notification/ntfnhandlers.go

```diff
@@ -135,7 +134,10 @@ func (q *collectionQueue) processBlock(bh *blockHashHeight) {
 
 	// Run synchronous block connected handlers in order.
 	for _, h := range q.syncHandlers {
-		h(&hash)
+		if err := h(&hash); err != nil {
```

Comment:
These block connected are executed in the defined order, synchronously, so if one fails we do not want to run the others.  Just log the error and bail.